pkg/volume/scaleio/sio_volume.go

```diff
@@ -448,19 +452,20 @@ func (v *sioVolume) setSioMgrFromConfig() error {
 	if v.sioMgr == nil {
 		configData := v.configData
 		applyConfigDefaults(configData)
+
+		configData[confKey.volSpecName] = v.volSpecName
```

Comment:
is there a reason this isn't setting these:
```
configData[confKey.secretName] = v.secretName
configData[confKey.secretNamespace] = v.secretNamespace
```

Reply:
@liggitt yes, secret and namespace comes from `options.Prameters=v.configData` during `NewProvisioner`.  But, will change code to make that clearer.